python/cudf/cudf/core/index.py

```diff
@@ -894,22 +892,12 @@ def _binaryop(
             return ret.values
         return ret
 
-    @_cudf_nvtx_annotate
+    # Override just to make mypy happy.
+    @cudf_nvtx_annotate
```

Comment:
Think this should be:

```suggestion
    @_cudf_nvtx_annotate
```

Reply:
Thanks, this is what I get for trying to just edit on the Github UI.